dashboard/app/jobs.go

```diff
@@ -497,7 +500,13 @@ func createJobResp(c context.Context, job *Job, jobKey *db.Key) (*dashapi.JobPol
 	if err != nil {
 		return nil, false, err
 	}
-	kernelConfig, _, err := getText(c, textKernelConfig, build.KernelConfig)
+
+	var kernelConfig []byte
```

Comment:
To avoid getText logic duplication:
```
configRef := job.KernelConfig
if configRef == 0 {
  configRef = build.KernelConfig
}
kernelConfig, _, err := getText(c, textKernelConfig, job.KernelConfig)
```